src/services/configService.ts

```diff
@@ -184,82 +208,56 @@ export class ConfigService {
         || tenantId,
       appId: this.getOption(constants.variableKeys.appId)
         || process.env.AZURE_CLIENT_ID
-        || appId
+        || appId,
     }
     config.provider.resourceGroup = (
       this.getOption("resourceGroup", config.provider.resourceGroup)
     ) || AzureNamingService.getResourceName(options);
 
-    // Get property from `runFromBlobUrl` for backwards compatibility
-    if (deployment && deployment.external === undefined && deployment["runFromBlobUrl"] !== undefined) {
-      deployment.external = deployment["runFromBlobUrl"];
+    const functionRuntime = this.getFunctionRuntime(runtime);
+    if (functionRuntime.language === SupportedRuntimeLanguage.PYTHON && os !== FunctionAppOS.LINUX) {
```

Comment:
can refactor to use `isPythonTarget` written above

Reply:
Actually, the `isPythonTarget` is not an option here. The `initializeConfig` runs first, and `isPythonTarget` depends on this having run first